### ISSUE TITLE
Demo QA - Homepage

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#83]
**Homepage**
https://demo.digitalgov.gov/index.html

**Keyboard tabbing**
ok until I hit articles; no dashed outline around featured images; is on 'read more' button and title

There used to be dashed outline around the Search button at top right, but, I see is missing on live and demo sites

**Events**
tab order goes thru each event ok, then to View All Events, which is above the events; it should be below.

Has 3 events for this month; live only has 2 -- should Janelle/CC add to live site so ppl can register?

**Programs & Services**
no dashed outline around featured images; is on 'read more' button and title

If still using this layout come February '18, the Sites block will have to be replaced with something else

---
Also, we removed Subscribe info & box from above footer last year because of HubSpot's send limit.. return?

**Comment by thisisdano** [comment]
@ToniBonittoGSA Thanks for this. I _think_ I see the keyboard tabbing (and dotted highlight) issues and the `view all events` issue on digitalgov.gov right now. I think there is some confusion around having multiple targets per issue link, but I'd like to hold on fixing anything until after the transition. It's maybe counterintuitive, but I'd like (as much as possible) to transition warts and all, as closely to the existing state as possible, and only make improvements once the new site is in place. So, I'll keep this issue open, but wait until later to address these things. OK?

And the Subscribe box stuff is all new to me. Is something we want to reinstate once the site is over on Federalist?

**Comment by ToniBonittoGSA** [comment]
Gotcha, sounds like a plan 👍

**Comment by jeremyzilar** [comment]
@ToniBonittoGSA I am closing. Feel free to re-open a new issue if there are things here that are outstanding bugs.
